# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: browser/actors/moz.build
# Commit: f64c8c158e52
# Full Hash: f64c8c158e52c96705cf1d507b9b30fe31305bd8
# Author: Mike Conley <mconley@mozilla.com>
# Date: 2020-04-23 09:52:48
# Description:
#   Bug 1631611 - Use a JSWindowActorChild to load about:newtab scripts from the ScriptPreloader rather than an observer. r=pdahiya
#   
#   This shifts the responsibility for ensuring that an about:home, about:welcome
#   or about:newtab page is where we inject the scripts to the JSWindowActor system,
#   rather than using the observer service on document creation and checking the
# ==============================================================================

diff -r af6964d3761b -r f64c8c158e52 browser/actors/moz.build
--- a/browser/actors/moz.build	Wed Apr 22 18:05:44 2020 +0000
+++ b/browser/actors/moz.build	Wed Apr 22 21:33:12 2020 +0000
@@ -26,6 +26,7 @@
     BUG_COMPONENT = ("Firefox", "Site Permissions")
 
 FINAL_TARGET_FILES.actors += [
+    'AboutNewTabChild.jsm',
     'AboutReaderChild.jsm',
     'BlockedSiteChild.jsm',
     'BlockedSiteParent.jsm',